[WIP] Logs support for sumologic k8s collection (#50)
SumoLogic/opentelemetry-collector-contrib: processor/fluentbitk8sprocessor/README.md

> ## <a name="fluentbitk8sprocessor"></a>Fluent bit k8s rocessor

Reviewer: Is this processor specific to sumologic-kubernetes-collection?

Author: So in order to perform metadata on the logs from the fluentbit from the k8s we need to apply that plugin. I believe it can be useful for the community too, eventually the functionality can be moved to the fluentforwarder, but I think it's better as separated processor

> ## <a name="fluentbitk8sprocessor"></a>Fluent bit k8s rocessor

Reviewer 2: 1) 

```suggestion
## <a name="fluentbitk8sprocessor"></a>Fluent bit k8s processor
```

2) 
> `<a name="fluentbitk8sprocessor"></a>`

Do we need this?

Author: > Do we need this?

I use this convention as it's used across the otelcol repository

> Processor which convert fluentd tag to the k8s metadata:

Reviewer 2: ```suggestion
Processor which converts fluentd tags to the following k8s metadata:
```

 - `pod_name`
 - `namespace`
 - `container_name`
 - `docker_id`

>  This plugin is required to correct work of k8s metadata

Reviewer 2: ```suggestion
 This plugin is required for correct work of k8s metadata
```
